tsconfig.json

```diff
@@ -1,13 +1,11 @@
 {
   "compilerOptions": {
     "target": "es2018",
-    "module": "commonjs",
+    "module": "ESNext",
```

Comment:
Is `ESNext` what Rollup expects, or does that introduce any constraints on supported node.js versions? Or does it not matter, as long as the TypeScript source files don't use any shiny new features?

Reply:
This is what rollup needs.

Reply:
Target allows to "dumb down" features.